x-pack/plugins/infra/server/lib/adapters/nodes/constants.ts

```diff
@@ -11,5 +11,5 @@ export const NODE_REQUEST_PARTITION_FACTOR = 1.2;
 export const NAME_FIELDS = {
   [InfraNodeType.host]: 'host.name',
   [InfraNodeType.pod]: 'kubernetes.pod.name',
-  [InfraNodeType.container]: 'docker.container.name',
+  [InfraNodeType.container]: 'container.name',
```

Comment:
I see that `kubernetes.pod.name` did not change, so the source issue probably only exists for the container.name.